lib/hanami/cli/interactive_system_call.rb

```diff
@@ -13,13 +13,12 @@ def initialize(out: $stdout, err: $stderr, exit_after: true)
         @out = out
         @err = err
         @exit_after = exit_after
-        super()
```

Comment:
This class doesn't inherit from anything, so this is a bit superfluous. I thought maybe this was designed to inherit from `SystemCall` (and I think there's a pretty decent overlap, so that seems like a reasonable refactoring), but this is only conjecture.